CONTRIBUTING.md

```diff
@@ -112,3 +151,4 @@ You can also join a `#daml-contributors` channel on our Slack: [damldriven.slack
 # Thank you!
 
 Thank you for taking the time to contribute!
+
```

Comment:
Seems unnecessary.

Reply:
Yeah, thanking people is always a waste of time.

...

Oh, you meant the extra blank line? 🤔